packages/web-pkg/src/composables/actions/helpers/useIsResourceNameValid.ts

```diff
@@ -0,0 +1,60 @@
+import { Resource } from '@opencloud-eu/web-client'
+import { useGettext } from 'vue3-gettext'
+import { RESOURCE_MAX_CHARACTER_LENGTH } from '../../../constants'
+import { useResourcesStore } from '../../piniaStores'
+
+export const useIsResourceNameValid = () => {
+  const { $gettext } = useGettext()
+  const resourcesStore = useResourcesStore()
+
+  const isFileNameValid = (
```

Comment:
```suggestion
  const isResourceNameValid = (
```

as it can be as well folders